storybook/cypress/e2e/modal.cy.ts

```diff
@@ -29,6 +28,16 @@ describe("Modal", () => {
     cy.wait(200).matchImageSnapshot();
   });
 
+  it.only("should have the propper role set by modal component", () => {
+    cy.get("dso-modal")
+      .invoke("attr", "component-role", "alert")
```

Comment:
Waarom niet een invoke op "prop"?

Reply:
Wat is het verschil?

Reply:
fixed

Reply:
Ook hier zie ik geen aanpassing. Heb je je commit niet gepushed?

Reply:
Correct, er was een merge conflict